spec/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.spec.js

```diff
@@ -320,6 +320,110 @@ describe('GOVUK.analyticsGa4.PIIRemover', function () {
     })
   })
 
+  function generatePhoneNumber (prefix, amount, suffix = '') {
+    var phoneNumber = prefix
+    for (var i = 0; i < amount; i++) {
+      phoneNumber += Math.floor(Math.random() * 10)
+    }
+    return phoneNumber + suffix
```

Comment:
I have a slight nervousness around using `+` here. You're obviously passing all strings in these tests so the result gets cast to a string, which is fine, but maybe it would be worth explicitly making sure this is a string to help future tests from hitting an issue? Something like `return "${phoneNumber}${suffix}"`?